tests/zexperimental/test_mcmc.py

```diff
@@ -273,6 +300,7 @@ def test_CWMH_regression_warmup(target: cuqi.density.Density):
     cuqi.experimental.mcmc.ULANew(cuqi.testproblem.Deconvolution1D().posterior, scale=0.0001),
     cuqi.experimental.mcmc.MALANew(cuqi.testproblem.Deconvolution1D().posterior, scale=0.0001),
     cuqi.experimental.mcmc.LinearRTONew(cuqi.testproblem.Deconvolution1D().posterior),
+    # cuqi.experimental.mcmc.UGLANew(create_lmrf_prior_target(dim=16)) #TODO: this cause a error in MHNew
```

Comment:
Please rebase on main, then if needed, update the scale in MHNew in checkpoint_targets to e.g. scale=0.0001. Should fix the issue.

Reply:
Done as suggested